charts/victoria-metrics-distributed/README.md

```diff
@@ -0,0 +1,283 @@
+# Victoria Metrics Helm Chart for Setting up VMCluster on Multiple Availability Zones
+
+![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)  ![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square)
+[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/victoriametrics)](https://artifacthub.io/packages/helm/victoriametrics/victoria-metrics-distributed)
+[![Slack](https://img.shields.io/badge/join%20slack-%23victoriametrics-brightgreen.svg)](https://slack.victoriametrics.com/)
+
+A Helm chart for Running VMCluster on Multiple Availability Zones
+
+## Prerequisites
+
+* Install the follow packages: ``git``, ``kubectl``, ``helm``, ``helm-docs``. See this [tutorial](../../REQUIREMENTS.md).
+
+* PV support on underlying infrastructure.
+
+* Multiple availability zones.
+
+## Chart Details
```

Comment:
Thanks for writing the doc! But I find it lacking a bit of details on the topology. Maybe explaining failure scenarios and how this topology helps to provide availability will be more understandable for reader.